recipes/docker.rb

```diff
@@ -216,6 +216,13 @@
             })
 end
 
+template node['hops']['docker']['fuse-seccomp-file'] do
```

Comment:
As far as I can see this is not a template, it is a file. Use the `cookbook_file` resource instead as we do for resourcemanager.yml for example